web/models.py

```
@@ -2790,3 +2790,106 @@ class ScheduledPost(models.Model):
+    description = models.TextField(blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    is_active = models.BooleanField(default=True)
+    start_date = models.DateField()
+    end_date = models.DateField(null=True, blank=True)
+    preferred_study_times = models.JSONField(default=list, help_text="List of preferred study times in 24h format")
+    study_duration = models.IntegerField(default=30, help_text="Preferred study session duration in minutes")
+    break_duration = models.IntegerField(default=10, help_text="Preferred break duration in minutes")
+
+    class Meta:
+        ordering = ["-created_at"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)